bibliopixel/remote/control.py

```diff
@@ -165,9 +166,20 @@ def run_animation(self, name=None):
     def stop_animation(self, data):
         return self.run_animation()
 
+    def change_brightness(self, data):
+        if isinstance(data, str):
+            try:
+                data = int(data)
+            except:
+                return False, 'Invalid brightness value!'
```

Comment:
Why not just throw an exception here and have the parent class catch it?  That first `True` vs `False` is what you'd do in a language without exceptions!